go/worker/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"breakfast/app"
```

Comment:
As per above, I think breakfast has been served, eaten, and cleaned up. We're going on vacation now!